docker/packager/binary/build.sh

```diff
@@ -1,4 +1,26 @@
 #!/usr/bin/env bash
+
+set -e
+
+# In case of test hung it is convenient to use pytest --pdb to debug it,
+# and on hung you can simply press Ctrl-C and it will spawn a python pdb,
+# but on SIGINT dockerd will exit, so ignore it to preserve the daemon.
+trap '' INT
+dockerd --host=unix:///var/run/docker.sock --host=tcp://0.0.0.0:2375 --default-address-pool base=172.17.0.0/12,size=24 &
```

Comment:
Where do these values (`172.17.0.0` and `:2375`) come from?

Reply:
2375 is a default port for docker. 
this IP is semi-default IP pool for composing networks